Angular/angular-mentoring/src/app/course-item/course-item.component.ts

```diff
@@ -28,8 +28,11 @@ export class CourseItemComponent implements OnInit {
   }
 
   private _selectTileColor() {
-    const isInFuture = this.item.creationDate > new Date();
-    const isNewerThanAFortnite = (new Date().getTime() - this.item.creationDate.getTime()) < this._calculateDays(14);
+    if (this.item === null) return;
+    
+    const date = new Date(`${this.item.date}`);
```

Comment:
We could just pass `const date = new Date(this.item.date);` right?

Reply:
Apparently so, I swear that it yelled at me for a string type when I first tried to pass in the value. Updated.